hack/tools/internal/tilt-prepare/main.go

```diff
@@ -823,11 +839,14 @@ func prepareWorkload(name, prefix, binaryName, containerName string, objs []unst
 
 				container.LivenessProbe = nil
 				container.ReadinessProbe = nil
+
+				container.Resources = corev1.ResourceRequirements{}
```

Comment:
Q: Is this an intentional overwrite of resources defined by providers?

Reply:
Yes, while debugging a provider with resources limit set I got a couple of OOM.

Reply:
Wow. They must have a very low default configuration if you hit it by just debugging (I assume at low scale)